examples/js/loaders/DRACOLoader.js

```diff
@@ -0,0 +1,202 @@
+// Copyright 2016 The Draco Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+THREE.DRACOLoader = function(manager) {
+    this.manager = (manager !== undefined) ? manager :
+        THREE.DefaultLoadingManager;
+    this.materials = null;
+};
+
+const DracoModule = Module;
```

Comment:
Ah, yes I spotted that. Not ideal... I copied this from Google's DRACO threejs renderer example: https://storage.googleapis.com/demos.webmproject.org/draco/draco_loader_throw.html
It looks like `Module` is an artifact of their emscripten process for draco_decoder.js